vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -219,9 +217,6 @@ struct ReactOptions {
 
   ReactDevOptions DeveloperSettings = {};
 
-  //! Additional properties associated with the ReactOptions.
-  Mso::JSHost::PropertyBag Properties;
```

Comment:
We should consider removing the code for this if we have no more usages. We always have it in source history.

Reply:
Yes, it is removed in this PR.

Reply:
I'm not seeing the file and Github is showing only 125 lines removed in the PR. Did we forget to include it in the change?

Reply:
You are right: I removed them from the project, but did not delete them. I will do it now.